config/application.rb

```diff
@@ -0,0 +1,31 @@
+require_relative "boot"
+
+require "rails/all"
+
+# Require the gems listed in Gemfile, including any gems
+# you've limited to :test, :development, or :production.
+Bundler.require(*Rails.groups)
+
+module HelloReactBackend
+  class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 7.0
+
+
+
+    config.middleware.insert_before 0, "Rack::Cors" do
```

Comment:
- [ ] Here you have made **"Rack::Cors"** as a string, But it should not be a string, it should be like this **Rack::Cors**. Please fix this.  You can take a look [here](https://github.com/cyu/rack-cors) as well.